cross-modal-search/flows/flow-query.yml

```diff
@@ -43,7 +43,7 @@ pods:
     shards: $JINA_SHARDS
     uses_after: pods/merge_matches_sort_topk.yml
     remove_uses_ba: true
-  - name: join_all
-    uses: _merge_root
-    needs: [text_indexer, image_kv_indexer]
-    read_only: true
+#  - name: join_all
```

Comment:
Also delete those if we don't need them please

Reply:
Deleted